### ISSUE TITLE
Urate tophus/i

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5862]
**Preferred term label:**
Urate tophus

**Synonyms**
Urate tophi (plural)

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/9195521/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/138900

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Intradermal urate crystal deposits appeared as small, superficial, pustule-like, whitish lesions. The lesions may be the focus of inflammatory episodes with increasing pain, swelling, and erythema of the intradermal tophi.
PMID: 9195521

**Comment by pnrobinson** [comment]
added term
